docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -263,9 +267,48 @@ Alternatively, click **Delete policy** to delete the policy.
 Existing data is not deleted.
 Any agents assigned to a policy must be unenrolled or assigned to a different policy before a policy can be deleted.
 
+[discrete]
+[[add-custom-fields]]
+== Add custom fields
+
+Use this setting to add a custom field and value set to all data collected from the {agents} enrolled in an {agent} policy. This is useful when you want to identify or visualize all of the data from a group of agents, and possibly manipulate it downstream.
```

Comment:
```suggestion
Use this setting to add a custom field and value set to all data collected from the {agents} enrolled in an {agent} policy. 
Custom fields are useful when you want to identify or visualize all of the data from a group of agents, and possibly manipulate the data downstream.
```

Reply:
Applies Asciidoctor's one sentence per line recommendation. 
Replaces  some pronouns with more descriptive words for better SEO and easier parsing.